### PR TITLE
Remove old listings code

### PR DESCRIPTION
**Pull request by schwers**

### CODE REVIEW

**Comment by schwers** [comment]
:eyeglasses: @ajacksified 

Deploy plan is enable the one-line feature flag, let that sit ~ 30minutes to an hour, if nothing bad happens finish removing the old code with this

File: src/views/pages/listing.jsx [hunk 1]
```diff
@@ -288,9 +285,9 @@ class ListingPage extends BasePage {
           list='comments'
           hideSort={ true }
         />
-        <div className={ `listing-content ${!listingRedesign ? 'container' :''}` } key='container'>
+        <div className={ 'listing-content' } key='container'>
```
Comment (nramadas) [inline src/views/pages/listing.jsx]: you can drop the curlies

File: src/views/pages/listing.jsx [hunk 2]
```diff
@@ -298,16 +295,16 @@ class ListingPage extends BasePage {
...
             user={ user }
             token={ token }
             saveUpdatedText={ this.saveUpdatedText }
             single={ true }
-            winWidth={ this.props.ctx.winWidth }
+            winWidth={ constants.POST_DEFAULT_WIDTH }
             toggleEdit={ this.toggleEdit }
           />
-          <div className={ `${listingRedesign ? 'container' : ''}` }>
+          <div className={ 'container' }>
```
Comment (nramadas) [inline src/views/pages/listing.jsx]: same here

File: src/views/components/listings/Listing.jsx [hunk 3]
```diff
@@ -89,13 +89,13 @@ export default class NewListing extends BaseComponent {
     } = this.props;
 
     const compact = this.state.compact || this.props.forceCompact;
-    const shouldRenderPaginationButtons = shouldPage && postsAndComments.length;
+    const shouldRenderPaginationButtons = shouldPage && listings.length;
 
     return (
-      <div className={ 'NewListing' }>
+      <div className={ 'Listing' }>
```
Comment (nramadas) [inline src/views/components/listings/Listing.jsx]: curly brackets

**Comment by ajacksified** [comment]
:nail_care: what @nramadas said

this is exciting

**Comment by schwers** [comment]
:nail_care: to me, remove @new-compact-thumbnail-width

**Comment by schwers** [comment]
:haircut:

**Comment by ajacksified** [comment]
looks legit :fish:

**Pull request merged**